pychunkedgraph/io/protobuf/chunkEdges.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package test;
+
+message ChunkEdges {
+  message Edges {
+    bytes edgeList = 1;
+    bytes affinities = 2;
+    bytes areas = 3;
+  }
+
+  Edges inChunk = 4;
+  bytes crossChunk = 5;
```

Comment:
crossChunk can be `Edges` as well - the message elements are optional in proto3, so no problem if `affinities` or `areas` are omitted.